packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -396,7 +396,8 @@ abstract class AnimatedWidgetBaseState<T extends ImplicitlyAnimatedWidget> exten
 ///
 /// The [AnimatedContainer] will automatically animate between the old and
 /// new values of properties when they change using the provided curve and
-/// duration. Properties that are null are not animated.
+/// duration. Properties that are null are not animated. Its child and 
```

Comment:
Can you remove the whitespace at the end of this line to make the analyzer check happy?

Reply:
Done.